### DISCUSSION TITLE
NocoDB Plugins [Apr 27]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5593] [Discord Threads]
> 🙋‍♂️ **Brunov25**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### NocoDB does support webhook. For some, we have extended webhook support (Slack, Discord, Teams) in beta. You can explore about them here   
#### https://docs.nocodb.com/setup-and-usages/account-settings#app-store   
####    
#### No Marketplace as such. We will be happy to understand use case that you are exploring with NocoDB to assist you better.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1090031891522015233)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
